src/api/tap.rs

```diff
@@ -4,7 +4,6 @@ use std::{
 };
 
 use futures::{future::try_join_all, FutureExt};
-use itertools::Itertools;
```

Comment:
Nice work dropping itertools where we could use the standard library instead.

Reply:
Yeah, some of the stuff it does is very helpful, but uh... `.collect_vec()` vs `.collect::<Vec<_>>()` ain't enough. 😂